### ISSUE TITLE
Support all `typing.*` types listed as :py:data:

### ISSUE THREAD

**Issue by flying-sheep** [agronholm/sphinx-autodoc-typehints#94]
As you know, some types aren’t `py:class`es but `py:data`. Currently handled are `Any`, `AnyStr`, `Callable`, `Optional` and `Union`, but there are more, most importantly `Tuple`:

```console
$ python -m sphinx.ext.intersphinx https://docs.python.org/3/objects.inv | grep -P '^([^\s]|\s+typing\.).*'
...
py:data
#       typing.Any                               library/typing.html#typing.Any
#       typing.AnyStr                            library/typing.html#typing.AnyStr
#       typing.Callable                          library/typing.html#typing.Callable
        typing.ClassVar                          library/typing.html#typing.ClassVar
        typing.NoReturn                          library/typing.html#typing.NoReturn
#       typing.Optional                          library/typing.html#typing.Optional
        typing.Tuple                             library/typing.html#typing.Tuple
#       typing.Union                             library/typing.html#typing.Union
...
```

**Comment by agronholm** [comment]
Would you mind creating a PR to address this?

**Comment by flying-sheep** [comment]
Done

**Comment by flying-sheep** [comment]
Fixed in 3c9f4f38fce0003a102e0436b6b34f85829dab48
